portfolio/src/main/webapp/script.js

```diff
@@ -38,3 +38,12 @@ function makeResponsive() {
     currentElement.className = "topnav";
   }
 }
+
+/**
+ * Part of the tutorial for week 3; retrieve response from Servlet 
+ */
+async function getHelloUsingAsyncAwait() {
+  const response = await fetch('/data');
+  const hello = await response.text();
+  document.getElementById('hello-container').innerText = hello.substring(4, hello.length-5-1);
```

Comment:
Should add a comment to explain what these magic numbers are (4, 5, 1). Or move them into a constant with an explanatory name. 

https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants